pegasus_wrapper/scripts/example_workflow_builder.py

```diff
@@ -69,20 +65,28 @@ def example_workflow(params: Parameters):
             multiply_by_x,
             {
                 "input_file": multiply_input_file,
-                "ouput_file": multiply_output_file,
+                "output_file": multiply_output_file,
                 "x": 4,
             },
             depends_on=[],
         ),
         locator=Locator("multiply"),
     )
 
+
+    slurm_params = Parameters.from_mapping({
+        "partition": "scanvenge",
```

Comment:
I am surprised this did not fail:
```suggestion
        "partition": "scavenge",
```